edk2toollib/uefi/edk2/test_path_utilities.py

```diff
@@ -811,9 +811,15 @@ def test_get_relative_path_with_nested_packages(self):
         pp2_name = "PPTestPkg2"
         self._make_edk2_package_helper(folder_pp2_abs, pp2_name)
 
+        # Nested packages should raise an exception by default
         self.assertRaises(Exception, Edk2Path, folder_ws_abs, [folder_pp1_abs])
         self.assertRaises(Exception, Edk2Path, folder_ws_abs, [folder_pp1_abs, folder_pp2_abs])
 
+        # Nested packages should no longer raise an exception
```

Comment:
a brief comment here to say this this can be removed once the workaround feature is removed will make it easier in future.

Reply:
Added in PR commit https://github.com/tianocore/edk2-pytool-library/pull/199/commits/faf600ea5c0691f6f30c5ffdc9650750c245a90c